app/serializers/submission_serializer/sections/case_details.rb

```diff
@@ -35,6 +39,14 @@ def to_builder
       def kase
         @kase ||= crime_application.case
       end
+
+      def case_type
+        return kase.case_type unless kase.case_type == CaseType::APPEAL_TO_CROWN_COURT.to_s
+
+        return kase.case_type unless kase.appeal_financial_circumstances_changed == YesNoAnswer::YES.to_s
+
+        CaseType::APPEAL_TO_CROWN_COURT_WITH_CHANGES.to_s
+      end
```

Comment:
Needs testing/checking but this is more readable to me
```suggestion
      def case_type
        return CaseType::APPEAL_TO_CROWN_COURT_WITH_CHANGES.to_s if  appeal_to_crown_court_case? || financial_circumstances_changed?

       kase.case_type
      end
      
      def appeal_to_crown_court_case? 
      	kase.case_type == CaseType::APPEAL_TO_CROWN_COURT.to_s
      end 
      
      def financial_circumstances_changed?
      	kase.appeal_financial_circumstances_changed == YesNoAnswer::YES.to_s
      end 
```